aws/table_aws_eks_cluster.go

```diff
@@ -0,0 +1,192 @@
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			Hydrate:    getEksCluster,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listEksClusters,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the cluster.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "created_at",
+				Description: "The Time when the cluster was created.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "version",
+				Description: "The Kubernetes server version for the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "endpoint",
+				Description: "The endpoint for your Kubernetes API server.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "role_arn",
+				Description: "The Amazon Resource Name (ARN) of the IAM role that provides permissions for the Kubernetes control plane to make calls to AWS API operations on your behalf.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "resources_vpc_config",
+				Description: "The VPC configuration used by the cluster control plane.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "logging",
+				Description: "The logging configuration for the cluster.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "identity",
+				Description: "The identity provider information for the cluster.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "status",
+				Description: "The current status of the cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "certificate_authority",
+				Description: "The certificate-authority-data for the cluster.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "platform_version",
+				Description: "The platform version of your Amazon EKS cluster.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "tags_src",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("tags"),
+				Hydrate:     getEksCluster,
+			},
+			{
+				Name:        "tags",
+				Description: "A list of tags assigned to the table",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getEksCluster,
```

Comment:
We should try to keep all the JSON fields at end of the table, please change accordingly